Sources/CLI/Volume/VolumeCreate.swift

```diff
@@ -0,0 +1,63 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the container project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
+import ContainerClient
+
+extension Application.VolumeCommand {
+    struct VolumeCreate: AsyncParsableCommand {
+        static let configuration = CommandConfiguration(
+            commandName: "create",
+            abstract: "Create a volume"
+        )
+
+        @Argument(help: "Volume name")
+        var name: String
+
+        @Option(name: .long, help: "Volume driver name")
```

Comment:
Would it be easier to add it at the whatever point when we implement another driver instead of baking this in today? I'm not totally against building out the whole thing if we're certain this is the model we want, just asking the question.